src/aind_metadata_mapper/stimulus/camstim.py

```diff
@@ -4,18 +4,21 @@
 
 import datetime
 import functools
+from pathlib import Path
 
 import aind_data_schema
 import aind_data_schema.core.session as session_schema
-import np_session
 import pandas as pd
+import requests
 
 import aind_metadata_mapper.open_ephys.utils.constants as constants
 import aind_metadata_mapper.open_ephys.utils.naming_utils as names
 import aind_metadata_mapper.open_ephys.utils.pkl_utils as pkl
 import aind_metadata_mapper.open_ephys.utils.stim_utils as stim
 import aind_metadata_mapper.open_ephys.utils.sync_utils as sync
 
+NPEXP_ROOT = Path("//allen/programs/mindscope/workgroups/np-exp")
```

Comment:
It's not recommended to hard-code endpoints. It'll be better to make this configurable.